featurebyte/query_graph/sql/common.py

```
@@ -57,6 +57,8 @@ def quoted_identifier(column_name: str) -> Expression:
     -------
     Expression
     """
+    if column_name == "*":
+        return expressions.Star()
```

Is this still needed? I think it's better to not have this logic as it most likely means this function has been misused.

---

Should we raise an error in that case?

---

Yes I think we can raise an error.